.gitignore

```diff
@@ -58,7 +58,8 @@ chronograf/server/swagger_gen.go
 http/swagger_gen.go
 
 # Ignore TSM testdata binary files
```

Comment:
Smallest of nits but I would update this to say `Ignore TSM/TSI testdata binary files`